_posts/2022-03-02-getting-started-project-flotta.markdown

```diff
@@ -0,0 +1,70 @@
+Kubernetes API. And so to manage the edgedevices and workloads you need to have Kubernetes or OpenShift up and
+running. Unfortunatelly the `flotta` operator is not yet published on the operator hub, so we have to deploy it manually from the github repo.
+That is why we need to have the `golang` and `make` installed on our machine.
+
+Following are the prerequisites:
+
+```bash
+ - Kubernetes or OpenShift cluster running with kubectl configured
+ - Cert manager
+```
+
+`Flotta` requires the [cert manager](https://cert-manager.io/docs/) for TLS key management for webhooks, so if you don't have it up and running on your cluster,
+you can execute following command to install it:
+
+```bash
+$ kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.7.1/cert-manager.yaml
+$ kubectl wait --for=condition=Ready pods --all -n cert-manager --timeout=60s
+```
+
+To deploy the latest version of the `flotta` operator execute following command:
+
+```bash
+$ kubectl apply -f https://github.com/project-flotta/flotta-operator/releases/download/v0.0.1/flotta-operator.yaml
+```
+
+This will clone the source code of `flotta` operator including all the deployment files. Then we apply those files using kubectl
+configured on the current machine, which is integrated in the Makefile script.
+
+By default the `flotta` operator is deployed in `flotta` namespace.
+
+```bash
+$ kubectl get all -n flotta
+NAME                                                     READY   STATUS    RESTARTS   AGE
+pod/flotta-operator-controller-manager-b7758f7b8-gww7d   2/2     Running   0          18h
+
+NAME                                                         TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)             AGE
+service/flotta-operator-controller-manager                   ClusterIP   10.103.155.185   <none>        8888/TCP,8043/TCP   18h
+service/flotta-operator-controller-manager-metrics-service   ClusterIP   10.98.35.222     <none>        8443/TCP,8080/TCP   18h
+service/flotta-operator-webhook-service                      ClusterIP   10.96.199.135    <none>        443/TCP             18h
+
+NAME                                                 READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/flotta-operator-controller-manager   1/1     1            1           18h
+
+NAME                                                           DESIRED   CURRENT   READY   AGE
+replicaset.apps/flotta-operator-controller-manager-b7758f7b8   1         1         1       18h
+```
+
+Flotta operator deploy two CRDs. First CRD called [edgedeployments](https://github.com/project-flotta/flotta-operator/blob/main/config/crd/bases/management.project-flotta.io_edgedeployments.yaml), used to manage the workloads. The second second CRD called [edgedevices](https://github.com/project-flotta/flotta-operator/blob/main/config/crd/bases/management.project-flotta.io_edgedevices.yaml) used to manage the devices.
+
+```bash
+$ kubectl get crd | grep edge
```

Comment:
perhaps it is better to grep by `project-flotta.io` since edge isn't specific to this project.